templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ manufacturer }}</h1>
```

Comment:
The message 'You want delete {{ manufacturer }}' is missing a word. It should be 'You want to delete {{ manufacturer }}' for correct grammar.